public/app/plugins/datasource/prometheus/result_transformer.tsx

```diff
@@ -94,6 +128,46 @@ export function transform(
   return dataFrame;
 }
 
+function getText(
+  exemplar: Exemplar,
+  seriesLabels: PromMetric,
+  exemplarTraceIDDestination?: ExemplarTraceIDDestination
+) {
+  let traceIDComponent: any;
+  if (exemplarTraceIDDestination) {
+    const traceID = exemplar.labels[exemplarTraceIDDestination.name];
+    const tag = ReactDOMServer.renderToString(<Tag name={traceID} colorIndex={6} />);
+    const href = exemplarTraceIDDestination.url.replace('${value}', traceID);
+    const anchorElement = `<a href="${href}" rel="noopener" target="_blank">${tag}</a>`;
+    traceIDComponent = anchorElement;
+  }
+
+  const template = `
```

Comment:
Can't this be a react element?

Reply:
It can but in the end it will be sent as a string right now.

Reply:
Why as a string? Does the new Graph API accept only strings for this?

Reply:
it does https://github.com/grafana/grafana/blob/master/public/app/plugins/panel/graph3/plugins/ExemplarMarker.tsx#L54

Reply:
@dprokop is that to allow non react code to use this? Although I would assume that even in Angular code you could pass on a react element or not?

Reply:
It's to allow html as as far as I see the annotation can contain html. 

@zoltanbedi ExemplarMarker is a copy of an annotation marker. Don't get too attached to it and feel free to adjust the markup/code to your needs. I agree with @aocenas that this should probably be a react element rather than hardcoded markup.

Reply:
Change text on ExemplarMarker to component.